docusaurus.config.js

```diff
@@ -94,6 +94,39 @@ const config = {
         sidebarPath: './sidebars.js',
       },
     ],
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'api-publisher',
+        path: 'docs/api-publisher',
+        editUrl: ({ docPath }) =>
+          `https://github.com/ed-fi-alliance-oss/ed-fi-alliance-oss.github.io/tree/main/${docPath}/`,
+        routeBasePath: 'docs/api-publisher',
+        sidebarPath: './sidebars.js',
+      },
+    ],
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'ods-api',
+        path: 'docs/ods-api',
+        editUrl: ({ docPath }) =>
+          `https://github.com/ed-fi-alliance-oss/ed-fi-alliance-oss.github.io/tree/main/${docPath}/`,
+        routeBasePath: 'docs/ods-api',
```

Comment:
For anyone else looking at this pull request, when you run locally, you can access the versions this way:

* http://localhost:3000/docs/ods-api/whats-new/
* http://localhost:3000/docs/ods-api/6.2/whats-new/